codespell_lib/_codespell.py

```diff
@@ -697,10 +738,15 @@ def main(*args):
             return EX_USAGE
         build_ignore_words(ignore_words_file, ignore_words)
 
-    ignore_words_list = options.ignore_words_list or []
-    for comma_separated_words in ignore_words_list:
-        for word in comma_separated_words.split(','):
-            ignore_words.add(word.strip())
+    uri_regex = options.uri_regex or uri_regex_def
+    try:
+        uri_regex = re.compile(uri_regex)
```

Comment:
I don't know why codecov doesn't think you're testing this,

Reply:
Yeah, I don't know... AFAICT the test `cs.main('--uri-regex=(', std=True)` should be hitting this.

Reply:
It seems to be green here: https://codecov.io/gh/codespell-project/codespell/compare/bc01afd55a91556e21214722ea364b017d2f657e...a5402ec7696c7da5cf05f1d266ff6dd1b8203e6b/diff